WBooks/MainViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MainViewController.swift
+//  WBooks
+//
+//  Created by Nicolas Manograsso on 05/10/2018.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import Networking
+import ReactiveSwift
+import UIKit
+
+final class MainViewController: UIViewController {
+
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    let bookRepo = NetworkingBootstrapper.shared.createWBooksRepository()
```

Comment:
add the access modifiers (in this case, `private` for both properties). Also if a property is private we start it with `_`, so in this case it should be `_bookRepository` and `_books` or `_bookArray`